cirq-core/cirq/ops/linear_combinations.py

```diff
@@ -360,8 +360,8 @@ def _pauli_string_from_unit(unit: UnitPauliStringT, coefficient: Union[int, floa
 class PauliSum:
     """Represents operator defined by linear combination of PauliStrings.
 
-    Since PauliStrings store their own coefficients, this class
-    does not implement the LinearDict interface. Instead, you can
+    Since `cirq.PauliString`s store their own coefficients, this class
```

Comment:
I think it would be useful to add some examples of how to construct and use pauli sums in the docstring here.

Reply:
Added.